src/sys/unix.rs

```diff
@@ -1618,6 +1618,68 @@ impl crate::Socket {
         }
     }
 
+    /// Get the value of the `SO_RCVLOWAT` option on this socket.
+    ///
+    /// For more information about this option, see [`set_rcv_lowat`].
+    ///
+    /// [`set_rcv_lowat`]: crate::Socket::set_rcv_lowat
+    // TODO: add Redox support, but I'm not sure if Redox supports SO_RCVLOWAT
+    #[cfg(all(feature = "all", not(target_os = "redox")))]
+    pub fn rcv_lowat(&self) -> io::Result<u32> {
```

Comment:
Can we come up with more descriptive names for these functions?

Reply:
Maybe receive_lowat and send_lowat? Lowat stands for low water mark. maybe something with that?
Or we could add a get_ prefix which would not align with the naming conventions of these socket option getter functions, but would be more descriptive. You tell me, since I don't know how bad it is to break naming conventions in this repo.